src/globals.js

```diff
@@ -4,15 +4,21 @@
  * consumers to decide. This makes dependency management a lot easier but comes
  * with some risks which can be allieviated with unit tests.
  */
+import lodash from 'lodash';
+import bluebird from 'bluebird';
```

Comment:
Hmm... Yea, I completely see your point and will change it back.
However I still have a feeling that in most cases people don't want to specify these just want to get it up and running as fast as possible, and they would like to do these optimisations later, when they are actually painful. But maybe I am wrong 😛 

My problem with the current implementation that it makes it complicated to setup.
What do you think about extending the functionality of the `Container` constructor, so it can take these dependency informations as a config parameter and calling the configure from inside itself, and throwing an error if not provided. I feel the "UX" - at least for me - would be much better like this.